app/templates/partials/calendar/event/edit_event_details_tab.html

```diff
@@ -40,25 +40,22 @@
 </div>
 
 <div class="form_row">
-  <label for="location_type">Location Type:</label>
-  <select id="location_type" name="location_type" required>
-    <option value="" disabled selected>Type</option>
-    <option value="vc_url">VC URL</option>
-    <option value="address">Address</option>
-  </select>
-  <input type="text" name="location" placeholder="VC URL/Location">
+    <label for="location">Location </label>
+    <input type="text" id="location" name="location" placeholder="Location">
 </div>
 
-<div class="form_row textarea">
-  <textarea id="say" name="description" placeholder="Description"></textarea>
+<div class="form_row">
+    <label for="vc_link">VC link: </label>
+    <input type="text" id="vc_link" name="vc_link" placeholder="VC URL">
```

Comment:
Use dashes in HTML ids: vc-link instead of vc_link.

Reply:
This is really problematic.
I've looked this through and though you are right this is a problem it exists all over the project in different tags and affects many many files and not related to the purpose of this PR (fixing create event).
Maybe it's necessary to open a new PR to fix this throughout the project . @yammesicka What do you think?